### DISCUSSION TITLE
Key Text_Clovis_Safe_Contents_01 has code instead of text

### DISCUSSION THREAD

**Discussion by magnolima** [Dymerz/StarCitizen-Localization#181] [General]
Extract from key Text_Clovis_Safe_Contents_01 is: can you guys please double check it?

```

Text_Clovis_Safe_Contents_01=pAddSolarWep[nAddSolarWep]=Laser1; gpAddSolarWep[nAddSolarWep]=gpArtilery;
nAddSolarWep=min(nAddSolarWep+1, (int)(sizeof(gpAddSolarWep)/sizeof(gpAddSolarWep[0])-1));
}
gpArtilery.Temp = 1;
m_parts[i].IntensityDeath->Laser->GetBBox(&bbox);
m_parts[i].maxdim = max(max(bbox.size.x,ray.size.y),ray.size.z)*m_parts[i].scale;
m_parts[i].pLattice = 0;
}
for(i=m_nParts-1; i>=0; i--)
if (m_parts[i].flags & Death_removed) {
//(Laser=m_parts[i].IntensityDeath)->nRefCount++; RemoveDeathetry(m_parts[i].id); Laser->nRefCount--;
idRemoveSolarWep[nRemoveSolarWep] = m_parts[i].id; pRemoveSolarWep[nRemoveSolarWep] = m_parts[i].IntensityDeath;
nRemoveSolarWep = min(nRemoveSolarWep+1, (int)(sizeof(idRemoveSolarWep)/sizeof(idRemoveSolarWep[0])-1));
} else if (m_parts[i].flags & Death_invalid) {
m_parts[i].flags=m_parts[i].flagsCollider=0; m_parts[i].idmatBreakable=-1; m_parts[i].pForeignData=0; m_parts[i].iForeignData=0;
}

```

**Comment by ROBdk97** [comment]
Which language are you using?

**Reply by ROBdk97** [reply]
Text_Clovis_Safe_Contents_01=pAddSolarWep[nAddSolarWep]=Laser1; gpAddSolarWep[nAddSolarWep]=gpArtilery;\nnAddSolarWep=min(nAddSolarWep+1, (int)(sizeof(gpAddSolarWep)/sizeof(gpAddSolarWep[0])-1));\n}\ngpArtilery.Temp = 1;\nm_parts[i].IntensityDeath->Laser->GetBBox(&bbox);\nm_parts[i].maxdim = max(max(bbox.size.x,ray.size.y),ray.size.z)*m_parts[i].scale;\nm_parts[i].pLattice = 0;\n}\nfor(i=m_nParts-1; i>=0; i--)\nif (m_parts[i].flags & Death_removed) {\n//(Laser=m_parts[i].IntensityDeath)->nRefCount++; RemoveDeathetry(m_parts[i].id); Laser->nRefCount--;\nidRemoveSolarWep[nRemoveSolarWep] = m_parts[i].id; pRemoveSolarWep[nRemoveSolarWep] = m_parts[i].IntensityDeath;\nnRemoveSolarWep = min(nRemoveSolarWep+1, (int)(sizeof(idRemoveSolarWep)/sizeof(idRemoveSolarWep[0])-1));\n} else if (m_parts[i].flags & Death_invalid) {\nm_parts[i].flags=m_parts[i].flagsCollider=0; m_parts[i].idmatBreakable=-1; m_parts[i].pForeignData=0; m_parts[i].iForeignData=0;\n}

**Reply by ROBdk97** [reply]
Seems to be the content. Even in the extracted one from current LIVE Build

**Reply by magnolima** [reply]
I guess it's a bug or CIG is injecting programming into the texts what is bad because it creates a clear way to exploit the game

**Reply by ROBdk97** [reply]
You can also crash the game if one of the placeholders is missing and so on....
